test1.2/task1/PriorityQueue/PriorityQueue.cs

```diff
@@ -0,0 +1,86 @@
+﻿// <copyright file="PriorityQueue.cs" company="SPBU">
+// Copyright (c) Alexander Bugaev 2024. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+// </copyright>
+
+namespace PriorityQueue;
+
+/// <summary>
+/// Data structure that supports adding element with specific values and int priorities and
+/// getting the element with the highest priority.
+/// </summary>
+/// <typeparam name="T">Type of values contained in the queue.</typeparam>
+public class PriorityQueue<T>
+{
+    private Element? head;
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="PriorityQueue{T}"/> class.
+    /// </summary>
+    public PriorityQueue()
+    {
+    }
+
+    /// <summary>
+    /// Gets a value indicating whether the queue is empty.
+    /// </summary>
+    public bool IsEmpty { get => this.head is null; }
```

Comment:
```suggestion
    public bool IsEmpty => this.head is null;
```
Это то же самое, но короче